.env.example

```diff
@@ -7,12 +7,13 @@ PORT=3000
 SERVER_URL=http://localhost:3000
 DEFAULT_LOCALE=ru
 FILE_FIELD_ADAPTER=local
-GOOGLE_RECAPTCHA_CONFIG='{"SITE_KEY":"6LcPRvQaAAAAAJRyxsFIB4rP5VH036pFOkNH8lgh", "SERVER_KEY":"6LcPRvQaAAAAADn_h1440Es7fXIGD0E4lpXR_FyF"}'
+# nosemgrep: generic.secrets.gitleaks.generic-api-key.generic-api-key
```

Comment:
we don't need this line because you already deleted dev keys

Reply:
Semgrep not so smart. It recognise google config but not checking actual values. That means in order to make it "fixed" we have to put "# nosemgrep:" comment